common-ktx/src/main/java/co/nimblehq/common/extensions/ToastExt.kt

```diff
@@ -0,0 +1,68 @@
+package co.nimblehq.common.extensions
+
+import android.app.Activity
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.widget.TextView
+import android.widget.Toast
+import androidx.annotation.IdRes
+import androidx.annotation.LayoutRes
+import androidx.fragment.app.Fragment
+
+/**
+ * Provide extension functions to Activity/Fragment to capable of showing Toast in a shorter call
+ *
+ * @param message A message to display
+ * @param duration Duration time to display message
+ */
+fun Activity.toastShort(message: CharSequence, duration: Int = Toast.LENGTH_SHORT) {
+    Toast.makeText(this, message, duration).show()
```

Comment:
In my opinion, by creating the toast like this, we could not control to hide the toast. Should we create a toast instance and manage to hide it as well? 🤔

Reply:
@hoangnguyen92dn  Can you explain more use cases? I didn't face that issue before.